src/flow/journey.js

```diff
@@ -30,11 +30,7 @@ const options = userOpts => {
   if (typeof userOpts.session === 'function') {
     sessionProvider = userOpts.session;
   } else {
-    const cookie = Object.assign(
-      { domain: parseUrl(userOpts.baseUrl).hostname },
-      userOpts.session.cookie || {}
-    );
-    const sessionOpts = Object.assign({}, userOpts.session, { cookie });
+    const sessionOpts = Object.assign({ cookie: {} }, userOpts.session);
```

Comment:
This still won't allow custom cookies. If look at the file below: `src/session/index.js` on line 18 it sets the userCookies from here `userOpts.cookie` but you are setting `cookie` to always be an empty object.

Reply:
No if cookie is defined in userOpts.session it will persist, if not then it will set default to {}

Reply:
I have added a test to check this, could you re-review?